### PR TITLE
docs: clarify force_delete_syncs documentation

### PR DESCRIPTION
**Pull request by asdfryan**
image

### CODE REVIEW

File: openapi/v2/mgmt/paths/sync_configs@{sync_config_id}.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@ put:
       required: false
       schema: 
         type: boolean
+      description: If true, any syncs for any deleted objects will be cascadingly deleted for all customers with this sync config
```
Comment (tomkit) [inline openapi/v2/mgmt/paths/sync_configs@{sync_config_id}.yaml:26]: Can we make this a :::danger admonition

**Comment by tomkit** [comment]
Also, there are a few places where the default is to do a cascade delete, can we add a :::danger admonition for those endpoints too

**Comment by asdfryan** [comment]
> Also, there are a few places where the default is to do a cascade delete, can we add a :::danger admonition for those endpoints too

where else do we do a cascade delete?

File: openapi/v2/mgmt/paths/sync_configs@{sync_config_id}.yaml (hunk 1 above)
Comment (asdfryan) [reply openapi/v2/mgmt/paths/sync_configs@{sync_config_id}.yaml:26]: apparently not. but I added it to the top level description

**Review by lucasmarshall — APPROVED** [review]

**Pull request merged**